test/doc/core_3_timeouts.cpp

```diff
@@ -283,7 +283,7 @@ void do_async_echo (basic_stream<Protocol, Executor>& stream)
 
 //]
 
-#if BOOST_BEAST_ENABLE_STACKFUL
+#if BOOST_BEAST_ENABLE_STACKFUL_TESTS
```

Comment:
If you leave this in, it will mess up the docs. Suggested:
```
#if defined(BOOST_BEAST_DOCS) || defined(BOOST_BEAST_ENABLE_STACKFUL_TESTS)
```

(The macro might be `BOOST_BEAST_DOXYGEN`, I can't recall)

Reply:
I should be able to remove it completely now